thirdparty/runtime_environments.go

```diff
@@ -58,3 +60,56 @@ func (c *RuntimeEnvironmentsClient) getImageNames(ctx context.Context) ([]string
 	}
 	return filteredImages, nil
 }
+
+// ImageDiffOptions represents the arguments for getImageDiff. AMI is the starting AMI, and AMI2 is the ending AMI.
+type ImageDiffOptions struct {
+	AMI   string
+	AMI2  string
+	Page  int
+	Limit int
```

Comment:
I think we discussed not supporting pagination for this route just because of the filtering at the end of the function